bait-files/src/main/java/com/google/LoginFilter.java

```diff
@@ -58,31 +53,30 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
       }
     }
     
-    // Case: User is logged in
-    else {
+    // Case: User is logged in but not registered (will replace else if with code checking registered)
+    else if (1!=1){
         /* TODO: Add case when user is not registered */
     }
 
-    // Is logged in and registered but is trying to access profile jsp or now restricted register
-    // page
-    if (request.getRequestURI().endsWith("profile.jsp")
-        || request.getRequestURI().endsWith("register.jsp")
-        || request.getRequestURI().endsWith("login.jsp")) {
-      RequestDispatcher requestDispatcher = request.getRequestDispatcher("WEB-INF/profile.jsp");
-      requestDispatcher.forward(request, response);
-      return;
+    else {
+    
+        // Is logged in and registered but is trying to access profile jsp or now restricted register
+        // page
```

Comment:
For now, the if-else statement only tells us whether the user is logged in or not, so this comment shouldn't say that we know the user is registered.. we only know that they are logged in.  Then I'd say adjust the comment to talk about being registered when you come back and implement the registration check =)

Reply:
Added more appropriate functionality